Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -1240,4 +1264,12 @@ getTripDetails state = {
   , onAnimationEnd : NoAction
   , backgroundColor : Color.white900
   , enablePaddingBottom : true
-}
\ No newline at end of file
+}
+
+driverPickUpStatusText :: DriverInfoCardState -> String -> String
+driverPickUpStatusText state eta = 
+  case state.props.zoneType of
+    SPECIAL_PICKUP -> "is at pickup location"
```

Comment:
Add localised text for "is at pickup location"

Reply:
@ashishsingh101